setup.py

```diff
@@ -306,6 +306,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
     "sphinx-argparse>=0.1.13",
     "sphinx-autoapi>=2.0.0",
     "sphinx-copybutton",
+    "sphinx-design",
```

Comment:
Shouldn't we pin this to at least a minimum version?

Reply:
Yeah. We might give a little helping hand to `pip` to get it resolve dependencies faster. 

Generally for docs we care little less than for "airflow" dependencies - because we will always have "latest matching" for those. Looking at this:

https://github.com/apache/airflow/actions/runs/7081768640/job/19271539099?pr=36041#step:6:18443

The version that is automatically installed is `0.5.0` (latest) so `>=0.5.0` would be fine.

Reply:
🤦 Good catch! I forgot to update this after testing and iterating.

Reply:
Updated here: https://github.com/apache/airflow/pull/36041/commits/d566a4003f942ea727e8c9555c8cf2001520351d